audformat/core/database.py

```diff
@@ -208,6 +209,34 @@ def segments(self) -> pd.MultiIndex:
         assert isinstance(index, pd.MultiIndex)
         return index.drop_duplicates()
 
+    def contains_nonunique_path(
+        self,
+    ) -> bool:
+        r"""Check if all paths are specified in a unique way.
```

Comment:
But isn't an absolute path also unique?

Reply:
No, as it can also be replaced by the relative path + `db.root`.

But I'm very open if you find a better way for the method.

Reply:
Maybe `db.is_portable()`?

Reply:
But if you think `contains_unique_files()` is clear, then I'm also fine with it.

Reply:
OK, I changed to `is_portable()` and updated the description and title of this pull request.